coordinator/src/admin.rs

```diff
@@ -226,71 +233,6 @@ impl From<lightning::chain::channelmonitor::Balance> for ChannelBalance {
     }
 }
 
```

Comment:
Hmm, in theory we could have removed a lot of this earlier.

I was motivated to remove it with this patch because I didn't want to spend any time migrating `ln_dlc_node::ChannelDetails` from `bitcoin:0.29` to `bitcoin:0.30`.